language/ir-testsuite/tests/block/block_prologue.mvir

```diff
@@ -27,19 +27,6 @@ main() {
     return;
 }
 
-//! new-transaction
-//! sender: vivian
-import 0x0.LibraBlock;
-import 0x0.LibraTimestamp;
-
-main() {
-    LibraBlock.block_prologue(1, 10, h"aa", {{vivian}});
```

Comment:
this is removed because we can't express a vector in the test currently

Reply:
Ah for this case, you can do something like this:
```
let v = Vector::empty<address>();
Vector::push_back<address>(&mut info, 0x1);
```

Reply:
You can do `import 0x0.Vector;` above and  ` LibraBlock.block_prologue(1, 10, Vector.empty<address>(), {{vivian}});`  here.